docs/GETTING_STARTED_GUIDE.md

```diff
@@ -95,6 +95,16 @@ rpm -ivh <rpm_file_name>
 
 5. After deployment, application data will be stored on the system under `/var/lib/rancher/k3s/storage/`.
 
+## Other Configuration Settings
+
+Some settings are not stored in the karavi-config-secret but in the csm-config-params ConfigMap, such as LOG_LEVEL and LOG_FORMAT. To update the karavi-authorization logging settings during runtime, run the below command on the K3s cluster, make your changes, and save the updated configmap data.
+
+```
+k3s kubectl -n karavi edit configmap/csm-config-params
+```
+
+This edit will not update the logging level for the sidecar-proxy containers running in the csi-driver pods. To update the sidecar-proxy logging levels, you must update the associated csi-driver ConfigMap in a similar fashion.
```

Comment:
Let's give details on how to update the log levels instead

Reply:
Updated with a short blurb using PowerFlex as an example.